lib/Bio/EnsEMBL/DataCheck/Checks/DisplayXrefExists.pm

```diff
@@ -53,11 +53,12 @@ sub tests {
   foreach my $type ("gene", "transcript") {
     my $desc = "${type}s have names set via display_xref_id";
     my $sql  = qq/
-      SELECT COUNT(*) FROM $type t
+      SELECT * FROM $type t
```

Comment:
Hi @JackCurragh, this data check looks good as I see no need to change sql query . 
test `is_rows_nonzero` is equivalent to `cmp_rows($dbc, $sql, '>', 0, $test_name)`  datacheck pass if row count is greater than zero .
This data check is advisory, so no impact on the handover service even if this fails.